libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/FairnessMetricTable.tsx

```diff
@@ -0,0 +1,171 @@
+          x: metricIndex,
+          y: 0
+        } as IFairnessMetricPointOptionsObject;
+      })
+      .concat(
+        this.props.fairnessStats.map((metricFairnessStats, metricIndex) => {
+          return {
+            color: "transparent",
+            max: metricFairnessStats.max,
+            maxCohort: metricFairnessStats.maxCohortName,
+            min: metricFairnessStats.min,
+            minCohort: metricFairnessStats.minCohortName,
+            value: Number(metricFairnessStats.ratio?.toFixed(3)),
+            x: metricIndex,
+            y: 1
+          } as IFairnessMetricPointOptionsObject;
+        })
+      );
+
+    return (
+      <HeatmapHighChart
+        configOverride={{
+          chart: {
+            height: 200,
+            type: "heatmap"
+          },
+          legend: {
+            enabled: false
+          },
+          series: [
+            {
+              borderWidth: 1,
+              data: items,
+              dataLabels: {
+                color: "black",
+                enabled: true
+              },
+              name: "Metrics",
+              type: "heatmap"
+            }
+          ],
+          title: {
+            align: "left",
+            text: this.props.title
+          },
+          tooltip: {
+            formatter() {
+              // to avoid semantic error during build cast point to any
+              const point = this.point as any;
+              const pointValue = point.value;
+              if (
+                point.x === undefined ||
+                point.y === undefined ||
+                pointValue === undefined ||
+                pointValue === null
+              ) {
+                return undefined;
+              }
+
+              return localization.formatString(
+                point.y === 0
+                  ? localization.ModelAssessment.ModelOverview
+                      .tableDifferenceTooltip
+                  : localization.ModelAssessment.ModelOverview
+                      .tableRatioTooltip,
+                // make metric name lower case in sentence
+                this.series.xAxis.categories[point.x].toLowerCase(),
+                pointValue,
+                point.min.toFixed(3),
+                point.minCohort,
+                point.max.toFixed(3),
+                point.maxCohort
+              );
+            }
+          },
+          xAxis: {
+            categories: columns,
+            opposite: true
+          },
+          yAxis: {
+            categories: [
+              localization.ModelAssessment.ModelOverview
+                .fairnessMetricDifference,
+              localization.ModelAssessment.ModelOverview.fairnessMetricRatio
+            ],
+            grid: {
+              borderWidth: 2,
+              columns: [
+                {
+                  labels: {
+                    formatter() {
+                      return `<div style='width:300px'><p>${this.value}</p></div>`;

```

Comment:
nit: should style be moved into styles file here?
```
style='width:300px'
```

Reply:
In general, I agree 100%.

In this particular case, you may have noticed that it's actually a string, not your usual React JSX. I briefly tried doing this with a style file, but it didn't work. I suspect that it has to do with this just being a string.